docs/ruleset_functions/get_zone_conditioning_category.md

```diff
@@ -107,10 +109,10 @@ Logic:
 
     - Else if zone has interior parking spaces, classify zone as unenclosed: `if（ space.lighting_space_type == "PARKING_AREA_INTERIOR" for space in zone.spaces ）: zone_conditioning_category_dict[zone.id] = "UNENCLOSED"`  
 
-    - Else if zone is crawlspace, classify zone as unenclosed: `else if ( ( zone.volume / sum( space.floor_area for space in zone.spaces ) ) < CRAWLSPACE_HEIGHT_THRESHOLD ) AND ( ( get_opaque_surface_type(surface) in ["HEATED SLAB-ON-GRADE", "UNHEATED SLAB-ON-GRADE"] ) AND ( surface.adjacent_to == "GROUND" ) for surface in zone.surfaces ): zone_conditioning_category_dict[zone.id] = "UNENCLOSED"`  (Note XC, need to check surface != Floor, RDS online not updated)
+    - Else if zone is crawlspace, classify zone as unenclosed: `else if ( ( zone.volume / sum( space.floor_area for space in zone.spaces ) ) < CRAWLSPACE_HEIGHT_THRESHOLD ) AND ( ( get_opaque_surface_type(surface) in ["HEATED SLAB-ON-GRADE", "UNHEATED SLAB-ON-GRADE"] ) AND ( surface.adjacent_to == "GROUND" ) for `**[CH: insert `any`]**` surface in zone.surfaces ): zone_conditioning_category_dict[zone.id] = "UNENCLOSED"`  (Note XC, need to check surface != Floor, RDS online not updated)
```

Comment:
I'll correct this.

Reply:
This is corrected. Also deleted the surface.adjacent_to requirement. Heated and unheated slab-on-grades are adjacent to ground by definition.